src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -169,14 +237,27 @@ export class SearchResults extends Component {
           </span>
           {resultsEndNumber} of {totalResults} results
         </h2>
+        {showYellowRibbonEnhancements && (
+          <AlertBox
+            content={TOOL_TIP_CONTENT}
+            headline={TOOL_TIP_LABEL}
+            isVisible={state.isSearchResultsTipOpen}
+            onCloseAlert={toggleAlertBoxAppearance(false)}
+            status="info"
+          />
+        )}
 
         {/* Table of Results */}
         <ul
           className="search-results vads-u-margin-top--2 vads-u-padding--0"
           data-e2e-id="search-results"
         >
-          {map(results, school => (
-            <SearchResult key={school?.id} school={school} />
+          {results.map((school, index) => (
```

Comment:
You may want to do `results?.map` if you don't want to use lodash here, otherwise if `results` is ever `undefined` it will throw a runtime error 🙂